mpi-proxy-split/test/Makefile

```diff
@@ -42,6 +42,7 @@ FILES=mpi_hello_world \
       Irecv_test Alloc_mem sendrecv_replace_test \
       Type_hvector_test Type_vector_test \
       ${FILES_FORTRAN} \
+      send_recv \
```

Comment:
It's minor, but why not keep `send_recv` and `send_recv_loop` on the same line.  Let's collapse this line into the next line.